src/super_gradients/training/transforms/obb/obb_random_rotate90.py

```diff
@@ -0,0 +1,81 @@
+import random
+from typing import Tuple, List, Dict
+
+import numpy as np
+from super_gradients.common.registry import register_transform
+from .obb_sample import OBBSample
+
+from .abstract_obb_transform import AbstractOBBDetectionTransform
+
+
+@register_transform()
+class OBBDetectionRandomRotate90(AbstractOBBDetectionTransform):
+    def __init__(self, prob: float = 0.5):
```

Comment:
I think there are some issues here...
This entire transform assumes bboxes_xyxy which is not consistent with what I see in the OBB sample.
Perhaps when you add a unit test lets pass some sample through all of these just see they run smoothly.

Reply:
That's a good catch! I think I copy-pasted this class from DetectionRandomRotate90 but actually never used and forgot it is present 🙈 For training I used Albumentation's random 90 rotation instead.
Removed in abc4e8d6d290b62ad3d1bdd008e59f8ca5180715